SupportClient/SupportClient/Views/FeedbackView.swift

```diff
@@ -0,0 +1,295 @@
+            typeDropdown.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -padding),
+            typeDropdown.leadingAnchor.constraint(equalTo: typeLabel.trailingAnchor, constant: 2 * padding),
+            typeDropdown.centerYAnchor.constraint(equalTo: typeLabel.centerYAnchor),
+            typeDropdown.heightAnchor.constraint(equalToConstant: 32)
+        ])
+        NSLayoutConstraint.activate([
+            messageLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: padding),
+            messageLabel.topAnchor.constraint(equalTo: typeLabel.bottomAnchor, constant: 2 * padding)
+        ])
+        NSLayoutConstraint.activate([
+            messageTextView.topAnchor.constraint(equalTo: messageLabel.bottomAnchor, constant: padding),
+            messageTextView.heightAnchor.constraint(equalToConstant: 100),
+            messageTextView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: padding),
+            messageTextView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -padding)
+        ])
+        NSLayoutConstraint.activate([
+            attachmentsLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: padding),
+            attachmentsLabel.topAnchor.constraint(equalTo: messageTextView.bottomAnchor, constant: 2 * padding)
+        ])
+        NSLayoutConstraint.activate([
+            addFileButton.leadingAnchor.constraint(equalTo: leadingAnchor, constant: padding),
+            addFileButton.topAnchor.constraint(equalTo: attachmentsLabel.bottomAnchor, constant: padding),
+            addFileButton.widthAnchor.constraint(equalToConstant: 100),
+            addFileButton.heightAnchor.constraint(equalToConstant: imageHeight)
+        ])
+        NSLayoutConstraint.activate([
+            attachmentsCollectionView.heightAnchor.constraint(equalTo: addFileButton.heightAnchor),
+            attachmentsCollectionView.leadingAnchor.constraint(equalTo: addFileButton.trailingAnchor, constant: padding),
+            attachmentsCollectionView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -padding),
+            attachmentsCollectionView.topAnchor.constraint(equalTo: addFileButton.topAnchor)
+        ])
+    }
+    
+    func checkStatus() {
+        if let _ = typeDropdown.selectedIndex {
+            if messageTextView.textColor != UIColor.lightGray || attachedFiles.count > 0 {
+                status = .complete
+            } else {
+                status = .incomplete
+            }
+        }
+    }
+    
+    // MARK: - OBJC Functions
+    @objc func dismissKeyboard (_ sender: UITapGestureRecognizer) {
+        self.endEditing(true)
+    }
+    
+    @objc func addFile(_ sender: UIButton) {
+        let imagePicker = ImagePickerController()
+        imagePicker.settings.theme.selectionStyle = .checked
+        imagePicker.settings.fetch.assets.supportedMediaTypes = [.image, .video]
+        currentViewController()!.presentImagePicker(imagePicker,
```

Comment:
couple of things:
1) we should avoid `!` and unwrap if anything
2) i don't think we should have this `currentViewController()` function because it can lead to unexpected & hidden behaviors (i.e. any view controller can add this as a subview w/o knowing that it could present an image picker). instead lets make a `FeedbackViewDelegate` that presents the image picker and then upon getting the attachedFiles, we can pass it to the FeedbackView again through a `configure(for attachedFiles)` function